src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/EmojiPicker.tsx

```diff
@@ -117,11 +117,15 @@ const EmojiPickerContainer: FC<EmojiPickerContainerProps> = ({
             }
           }}
         >
+          {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */}
           <div
             ref={emojiRef}
             style={{maxHeight: window.innerHeight, ...style}}
             role="dialog"
             data-uie-name="emoji-picker-dialog"
+            onClick={event => {
+              event.stopPropagation();
```

Comment:
We need to stop propogating emoji picker search input click event here, otherwise it bubbles up to the message element div where we have a handlefocus function which receives the click event and search input bar inside emoji picker looses focus.

Reply:
ooohhh that's the reason!!! Makes sense. 

Can you add this as a comment in the code, that could help for future references :)